src/components/app/project-gallery/card.tsx

```diff
@@ -11,6 +11,7 @@ import { ProjectUser } from '@/api/types/project-user';
 import { UserAuthHelper } from '@/helpers';
 import { Link } from 'gatsby';
 import { ApiResponse, ErrorResponse } from '@/api/types/responses';
+import { navigate } from '@reach/router';
```

Comment:
@aaron2212 `@reach/router` is not explicitly declared in our dependencies. Maybe get it from `gatsby` as per Gatsby documentation? :thinking:

```suggestion
import { navigate } from 'gatsby';
```